.github/workflows/check-and-test.yml

```diff
@@ -0,0 +1,79 @@
+# This action runs lint checks and tests against the code.
+name: Check and Test
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on pushes to master and pull requests for any branch
+  push:
+    branches: [master]
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  check-and-test:
+    name: Check and Test
+    # The type of runner that the job will run on
+    runs-on: ubuntu-16.04
```

Comment:
nit: I saw that in the Travis CI, too, but just updated it to `20.04`... Probably feel free to just upgrade it.

Reply:
Using `latest` broke CI in `connection-model` this is why I preemptively used the same version here. But the packages that were breaking the CI there are not used here, so I'll change it. I also did some digging in history and I think I'll remove it there also, seems like they were added as a workaround for some issue that is non-existent anymore